sample/src/main/java/com/github/stephenvinouze/materialnumberpickersample/MainActivity.kt

```diff
@@ -36,34 +35,33 @@ class MainActivity : AppCompatActivity() {
         }
         customButton.setOnClickListener {
             val numberPicker = MaterialNumberPicker(
-                    context = this,
-                    minValue = 1,
-                    maxValue = 50,
-                    value = 10,
-                    separatorColor = ContextCompat.getColor(this, R.color.colorAccent),
-                    textColor = ContextCompat.getColor(this, R.color.colorPrimary),
-                    textSize = resources.getDimensionPixelSize(R.dimen.numberpicker_textsize),
-                    textStyle = Typeface.BOLD_ITALIC,
-                    editable = false,
-                    wrapped = false,
-                    fontName = "Hand.ttf",
-                    formatter = NumberPicker.Formatter {value ->
-                        return@Formatter "Value $value"
-                    }
+                context = this,
+                minValue = 1,
+                maxValue = 50,
+                value = 10,
+                separatorColor = ContextCompat.getColor(this, R.color.colorAccent),
+                textColor = ContextCompat.getColor(this, R.color.colorPrimary),
+                textSize = resources.getDimensionPixelSize(R.dimen.numberpicker_textsize),
+                textStyle = Typeface.BOLD_ITALIC,
+                editable = false,
+                wrapped = false,
+                fontName = "Hand.ttf",
+                formatter = NumberPicker.Formatter { value ->
+                    return@Formatter "Value $value"
+                }
             )
             presentPickerInAlert(numberPicker, getString(R.string.alert_custom_title))
         }
     }
 
     private fun presentPickerInAlert(numberPicker: NumberPicker, title: String) {
         AlertDialog.Builder(this)
-                .setTitle(title)
-                .setView(numberPicker)
-                .setNegativeButton(getString(android.R.string.cancel), null)
-                .setPositiveButton(getString(android.R.string.ok)) { _, _ ->
-                    Toast.makeText(this, getString(R.string.picker_value, numberPicker.value), Toast.LENGTH_LONG).show()
-                }
-                .show()
+            .setTitle(title)
+            .setView(numberPicker)
+            .setNegativeButton(getString(android.R.string.cancel), null)
+            .setPositiveButton(getString(android.R.string.ok)) { _, _ ->
+                Toast.makeText(this, getString(R.string.picker_value, numberPicker.value), Toast.LENGTH_LONG).show()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/StephenVinouze/MaterialNumberPicker/blob/ae55545cc2574a05af41ab00efd15ced1c007f9e/sample/src/main/java/com/github/stephenvinouze/materialnumberpickersample/MainActivity.kt#L63"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>